addons/block_code/ui/blocks/statement_block/statement_block.tscn

```diff
@@ -32,7 +34,6 @@ layout_mode = 2
 mouse_filter = 1
 script = ExtResource("2_lctqt")
 color = Color(1, 1, 1, 1)
-outline_color = Color(0.8, 0.8, 0.8, 1)
```

Comment:
Ack. I should have resaved this scene when removing the `outline_color` property in cc307af93c21f39bf695f205f8731385e703708a.